auth-api/src/auth_api/services/products.py

```diff
@@ -53,7 +53,7 @@ def create_product_subscription(org_id, subscription_data: Tuple[Dict[str, Any]]
                 subscriptions_model_list.append(product_subscription)
             else:
                 raise BusinessException(Error.DATA_NOT_FOUND, None)
-            if subscription.get('product_roles') is not None:
+            if subscription.get('productRoles') is not None:
```

Comment:
Can we give all roles for that product by default? Else UI will need to query all the roles and then pass it.

Reply:
u mean if nothing is passed by default ? The only reason i put it is if we ever put the roles in UI as well..a role selection as well

Reply:
I added the new logic..Is that good ?
If no roles are passed from UI for a product , all roles gets added..
If the logic is okay , will add more tests..lmk